src/screens/ProfileScreen/index.js

```diff
@@ -171,7 +173,6 @@ const ProfileScreen = ({route}) => {
   const [isLoadingUpdateImageCamera, setIsLoadingUpdateImageCamera] = React.useState(false);
   const [errorChangeRealName, setErrorChangeRealName] = React.useState('');
   const [postOffset, setPostOffset] = React.useState(0);
-  const [isLastPage, setIsLastPage] = React.useState(false);
   const [loading, setLoading] = React.useState(false);
   const [isPostOptionModalOpen, setIsOptionModalOpen] = React.useState(false);
   const [selectedPostForOption, setSelectedPostForOption] = React.useState(null);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The state variable `isLastPage` has been removed. This might affect the logic of pagination if it was used to determine when to stop fetching more feeds. If the new implementation handles this differently, then it's fine. Otherwise, you might want to reconsider this change.

```diff
- const [isLastPage, setIsLastPage] = React.useState(false);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->